frontend/src/lib/apiClient.js

```diff
@@ -95,13 +95,22 @@ const createRecipe = recipe => apiClient.post('api/v1/recipes', {
     difficulty: recipe.difficulty,
     duration: recipe.duration,
     servings: recipe.servings,
+    tags: recipe.tags,
   },
 }, {
   headers: {
     Authorization: localStorage.jwt,
   },
 });
 
+// TAGS
+
+const getTags = () => apiClient.get('/api/v1/tags')
+  .then(results => results.data);
+
+const getRecipeTags = id => apiClient.get(`/api/v1/recipes/${id}/tags`)
```

Comment:
I think we may need to delete this function now that we're not using it (at least for now). It's causing CircleCI to fail due to linting errors:
```
./src/containers/RecipePage/Recipe.js
  Line 9:  'getRecipeTags' is defined but never used  no-unused-vars
```